crates/js-sys/src/lib.rs

```diff
@@ -2940,6 +2940,34 @@ extern "C" {
     #[wasm_bindgen(method, js_class = "String", js_name = endsWith)]
     pub fn ends_with(this: &JsString, search_string: &str, length: i32) -> bool;
 
+    /// The static String.fromCharCode() method returns a string created from
+    /// the specified sequence of UTF-16 code units.
+    ///
+    /// https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/fromCharCode
+    ///
+    /// # Notes
+    ///
+    /// There are a few bindings to `from_char_code` in `js-sys`: `from_char_code1`, `from_char_code2`, etc...
+    /// with different arities.
+    #[wasm_bindgen(static_method_of = JsString, js_class = "String", js_name = fromCharCode)]
+    pub fn from_char_code1(a: f64) -> JsString;
```

Comment:
These should be using `u32` parameter types.

`char_code_at` returns `NaN` for out of bounds indices, which is why it returns `f64`, but we don't have that issue here.